Makefile

```diff
@@ -51,6 +51,8 @@ CONTROLLER_TOOLS_VERSION ?= v0.12.0
 ALL_SRC := $(shell find . -name '*.go' -type f | sort)
 CW_AGENT_OPERATOR_IMPORT_PATH = "github.com/aws/amazon-cloudwatch-agent-operator"
 
+GOTEST_OPTS := $(shell go list ./... | grep -v integration-tests)
```

Comment:
We can ignore the integration-tests as part of `make test`

Reply:
nit: Could add an `integration-test` build tag to the integration tests, so they're skipped normally.

```go
//go:build integration-test
```